templates/jfrog-ami-creation.template.yaml

```diff
@@ -203,25 +199,27 @@ Resources:
         config-ansible-art-ami:
           files:
             /root/.jfrog_ami/jfrog-ami-setup.yml:
-              content: !Sub |
+              content: !Sub
+                - |
                   # Base install for JFrogAMIInstance
                   - import_playbook: artifactory-ami.yml
                     vars:
                       ami_creation: true
                       artifactory_version: ${ArtifactoryVersion}
-                      artifactory_ha_enabled: true
+                      artifactory_ha_enabled: false
```

Comment:
@vinayagg Should this be `true` or `false` for QS/MP? I now think it should be 'true' but not sure

Reply:
true